Sources/StreamChatUI/CommonViews/InputTextView/InputTextView.swift

```diff
@@ -36,6 +36,8 @@ open class InputTextView: UITextView, AppearanceProvider {
         }
     }
 
+    open var placeholderContainer: UIStackView = .init()
```

Comment:
Can we add a description of the goal of this view? 🙏

Reply:
Important not to slow down the CI today just for this though.